src/components/UI/Inputs/Select/index.tsx

```diff
@@ -0,0 +1,171 @@
+import {
+  FormField,
+  FormFieldProps,
+  Select as Input,
+  ThemeContext,
+  ThemeType,
+} from 'grommet';
+import PropTypes from 'prop-types';
+import * as React from 'react';
+import { css } from 'styled-components';
+
+import SearchInput from './SearchInput';
+
+const customTheme: ThemeType = {
+  global: {
+    drop: {
+      background: 'none',
+      shadowSize: 'none',
+      border: {
+        radius: '0',
+      },
+      zIndex: '9999',
+    },
+  },
+  select: {
+    background: 'none',
+    icons: {
+      color: { dark: 'text', light: 'text' },
+    },
+    // @ts-expect-error
+    searchInput: SearchInput,
+    container: {
+      // @ts-expect-error
+      extend: css`
```

Comment:
Just learned that we can use `styled-components`'s `css` function instead of a regular arrow function. The type definitions don't like it, but it works.